cypress/support/component.ts

```diff
@@ -1,21 +0,0 @@
-// Import commands.js using ES2015 syntax:
-import { mount } from 'cypress/vue'
-import './commands'
-
-// Augment the Cypress namespace to include type definitions for
-// your custom command.
-// Alternatively, can be defined in cypress/support/component.d.ts
-// with a <reference path="./component" /> at the top of your spec.
-
-declare global {
-  namespace Cypress {
-    interface Chainable {
-      mount: typeof mount
-    }
-  }
-}
-
-Cypress.Commands.add('mount', mount)
```

Comment:
question: why was this removed?

Reply:
Same as the file in `shared-ui-components`. It's
- not imported by other modules, so the `Cypress.Commands.add('mount', mount)` is not executed
- the `declare global {` now conflicts with the declarations in `cypress/support/index.d.ts`
- the `Cypress.Commands.add('mount', mount)` (if it's executed), will conflict with `cypress/support/commands.ts:32`

Reply:
@nekolab I forgot to respond here: Cypress automatically looks for this file (it's the default configuration).

In this repo, since we weren't really adding anything here, there's no harm so far in removing; however, in `konnect-ui-apps` for example, we have an entire component test configuration that differs from the e2e test setup, so the `component.ts` files there must remain.

Reply:
Thanks for the explanation! I think thats also why we have `cy.baseMount` in `konnect-ui-apps`.